lib/engine/share_pool.rb

```diff
@@ -251,9 +247,14 @@ def transfer_shares(bundle, to_entity,
 
       return unless bundle.partial?
 
+      handle_partial(bundle, self, owner)
```

Comment:
Several of the other places where this is added have a "return" afterwards. Are they redundant or should this have a return?

Reply:
Those other `returns` are control flow to break the execution of the function. This instance, you'll notice, is at the very end of the function, which already has an implicit `return`.

Reply:
Silly me, I saw the "end" and thought it was the close of a function, not a conditional, as the preceding lines were not visible..